plugin/pkg/scheduler/api/types.go

```diff
@@ -92,14 +92,14 @@ type PriorityArgument struct {
 	LabelPreference *LabelPreference
 }
 
-// Holds the parameters that are used to configure the corresponding predicate
+// ServiceAffinity holds the parameters that are used to configure the corresponding predicate
```

Comment:
ServiceAffinity holds the parameters that are used to configure the corresponding predicate in scheduler policy configuration.